src/main/java/com/scalar/db/benchmarks/tpcc/TpccLoader.java

```diff
@@ -161,27 +108,134 @@ static class Exclusive {
           .put(STOCK, STOCK_HEADER)
           .put(WAREHOUSE, WAREHOUSE_HEADER)
           .build();
-  private int endWarehouse;
+  private final DistributedTransactionManager manager;
+  private final int concurrency;
+  private final int startWarehouse;
+  private final int endWarehouse;
+  private final boolean skipItemLoad;
+  private final boolean useTableIndex;
+  @Nullable private String directory;
```

Comment:
We can make this field `final`:
```suggestion
  @Nullable private final String directory;
```

Reply:
Fixed in 2286bc9.